### ISSUE TITLE
Documentation: add ability to post images

### ISSUE THREAD

**Issue by jimkp** [kororaproject/website#56]
When creating a document there is no ability to include an image. Currently it is emailed to the webmaster and then he does it manually. This is an inefficient use of his time and discourages the use of images in documents.

The ability to upload and include an image while creating a document will simplify the process.

**Comment by jimkp** [comment]
this could be useful in News too

**Comment by dyoungwd** [comment]
could we not just have the image hosted else where and use markup / html to add an image - that would save having them uploaded to the server and admin having to manually

**Comment by jimkp** [comment]
Yes we could but that would mean the images would be scattered around the web in personal accounts. It would be better if they were in one place.

Actually I was considering doing that as a temporary thing until this was implemented. I just need to find a reliable place to store them.
